### PR TITLE
recipe: fix units of axonal velocity

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Review by pramodk — APPROVED** [review]
that would have too fast I guess :)

File: source/recipe.rst [hunk 1]
```diff
@@ -272,7 +272,7 @@ To assign synapse properties, the classification field needs to be set:
 Two optional attributes may be set:
 
 - ``neuralTransmitterReleaseDelay`` with a default of 0.1
-- ``axonalConductionVelocity`` with a default of 300 m/s
+- ``axonalConductionVelocity`` with a default of 300 μm/s
```
Comment (jdcourcol) [inline source/recipe.rst:275]: 300um/ms based on Srikanth's input

Comment (jdcourcol) [reply source/recipe.rst]: I am good. let's add Sirio and Natali to assess they are good with that.

**Review by seirios — APPROVED** [review]
Seems correct and consistent to me.

**Pull request merged**